Doc/library/argparse.rst

```diff
@@ -201,9 +201,10 @@ ArgumentParser objects
    * usage_ - The string describing the program usage (default: generated from
      arguments added to parser)
 
-   * description_ - Text to display before the argument help (default: none)
+   * description_ - Text to display before the argument help (default:
+     no text)
 
-   * epilog_ - Text to display after the argument help (default: none)
+   * epilog_ - Text to display after the argument help (default: no text)
```

Comment:
```suggestion
   * epilog_ - Text to display after the argument help (by default, no text)
```

I might suggest this form to further avoid implying this as the default value to this named _argument_, as opposed to the _behavior_ by default